go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.0
 	sigs.k8s.io/controller-tools v0.4.0
 )
+
+replace github.com/Azure/go-autorest/autorest => github.com/Azure/go-autorest/autorest v0.11.18
```

Comment:
Please leave a comment on this PR or in `go.mod` on why we needed this replacement, i.e., that the SDK library does not use go modules.

Reply:
Comment was left. `go get` was used to update the go-autorest dependency. Done!